google/cloud/storage/internal/async/connection_impl.cc

```diff
@@ -297,8 +302,14 @@ AsyncReaderConnectionFactory AsyncConnectionImpl::MakeReaderConnectionFactory(
                       google::storage::v2::ReadObjectRequest const& proto)
       -> future<StatusOr<std::unique_ptr<StreamingRpc>>> {
     ApplyQueryParameters(*context, *options, request);
+    auto timeout = ScaleStallTimeout(
+        options->get<storage::DownloadStallTimeoutOption>(),
+        options->get<storage::DownloadStallMinimumRateOption>(),
```

Comment:
Slightly interesting sub-text:  While `Options::get<>()` guarantees that you at least get a value-initialized duration, it is unclear (to me) that a zero "Rep" guarantees a zero duration.  I imagine it always does in practice though.

Reply:
Ack.